Utils/pom.xml

```diff
@@ -257,7 +257,7 @@
             <dependency>
                 <groupId>org.apache.hadoop</groupId>
                 <artifactId>hadoop-common</artifactId>
-                <version>2.7.1</version>
+                <version>2.2.0</version>
```

Comment:
><version>2.2.0</version> [](start = 16, length = 24)

down to 2.2?

Reply:
has to... since the current spark-core 2.0/2.1 dependent on 2.2.0